roles/keystone/defaults/main.yml

```diff
@@ -26,3 +29,7 @@ keystone:
   logging:
     debug: False
     verbose: True
+
+  ldap:
+    url: ldap://127.0.0.1:389
```

Comment:
LDAP is put into the haproxy, but then keystone is configured to hit it directly. What's the intent?